### ISSUE TITLE
The loss of training.

### ISSUE THREAD

**Issue by noseDewdrop** [noseDewdrop/112#1]
I had trained 2 days for that, but I found that the loss which has no trend of convergence oscillated for a long time.I reduced the learning rate from 1e-3 to 1e-5, but that got nothing. The test results are good but can not be better, There are some terrible things such as the proportion of good parts is only about 10%, anothers have the bad face structure, difference eyes and two mouths.

I'm waiting for  your help and instruction.
